Models/WeeklyGoals.model.ts

```diff
@@ -0,0 +1,5 @@
+export enum WeeklyGoals {
```

Comment:
enum name WeeklyGoal wouldn't be better?

Reply:
Sure, it'll be changed.